app/components/conferences/dayOfEvent/mainstage/videostreamer.js

```diff
@@ -10,6 +10,9 @@ export default function Videostreamer(props) {
     "Thanks for joining in! Please start the stream."
   );
 
+  const parentDevelopement = ["127.0.0.1", "localhost"];
+  const parentProduction = ["https://conf.rceng.shop"];
```

Comment:
Hey @VanshulB 
Thanks for the fix, but that's not how the scalability is handled. Think about the following use cases:
1. I am doing the development on a VM, which definitely will have `172.xx.yy.z` something like this IP instead of the `127.0.0.1`
2. For production, we are not the only users going to use RC4Conferences; suppose you have a community website (`community.vanshul.home`) and integrated this component into it, then this won't work.

Please think carefully about this cases. Thank you!

Reply:
Hey @Dnouv 

In order to handle scalability, we have used environment variables to set the domain name/ ip dynamically instead of hardcoding them into the codebase! have added the documentation for adding the environment variable if we are developing in a virtual machine or while using in the production environment we can specify the domain on which our website is gonna host. I hope that's what you wanted! Please tell any suggestions.
Thank you!